src/CMakeLists.txt

```diff
@@ -192,12 +192,6 @@ include(util/src.cmake)
 include(time/src.cmake)
 include(middleware/src.cmake)
 
-option(BUILD_POPOTO OFF)
```

Comment:
I add the nlohmann/json.hpp directly to goby3/src/util/thirdparty/nlohmann/json.hpp since it's a single file we can avoid dependency issues and always build the Popoto driver.